Tests/SnapshotTestingTests/SnapshotTestingTests.swift

```diff
@@ -188,9 +196,9 @@ class SnapshotTestingTests: SnapshotTestCase {
     #endif
     if #available(macOS 10.14, *) {
       label.text = "Hello."
-      assertSnapshot(of: .view(precision: 0.9), matching: label, named: platform)
+      assertSnapshot(matching: label, as: .view(precision: 0.9), named: platform)
```

Comment:
Think `.view` and `.viewController` and `.scene` should all probably be `.image`:

``` swift
assertSnapshot(matching: scene, as: .image(size: .init(width: 50, height: 50)))
assertSnapshot(matching: label, as: .image(precision: 0.9))
```

Or `.rendered`:

``` swift
assertSnapshot(matching: scene, as: .rendered(size: .init(width: 50, height: 50)))
assertSnapshot(matching: label, as: .rendered(precision: 0.9))
```